packages/help-center/src/components/back-button.tsx

```diff
@@ -12,11 +12,25 @@ export const BackButton = ( { onClick, backToRoot = false, className }: Props )
 	const location = useLocation();
 	const navigate = useNavigate();
 	const buttonClassName = clsx( 'back-button__help-center', className );
+	const queryParams = new URLSearchParams( location.search );
+	const backUrl = queryParams.get( 'backUrl' );
 
-	function defaultOnClick() {
-		if ( backToRoot ) {
+	function handleOnClick() {
+		const currentPath = location.pathname;
+		if ( currentPath === '/odie' ) {
 			navigate( '/' );
-		} else if ( location.key === 'default' ) {
+			return;
+		}
+
+		if ( backUrl ) {
+			navigate( backUrl );
+			return;
```

Comment:
Hey @AllTerrainDeveloper sorry it took me a while to get to this. Was there a reason you needed to do this? By default it should be going back to `-1` which is just the previous location.